### ISSUE TITLE
Bug with Id generator is causing accessibility errors

### ISSUE THREAD

**Issue by BLuEScioN** [JedWatson/classnames#213]
The bug I am seeing is that some tab and tab panel ids are being reused. This fails accessibility compliance standards. Not to mention that Ids in the DOM are supposed to be unique. My special use case of the tabs is that I use the control within itself, so a  tab has another tab control within it. I believe this is the source of the bug. Here is a snapshot of the DOM that shows the duplicate Ids being generated. 

![image](https://user-images.githubusercontent.com/20312835/81890523-06fc9300-955b-11ea-9195-3a1933e4d542.png)

**Comment by newyork-anthonyng** [comment]
@BLuEScioN `classnames` is a JavaScript utility that conditionally joins strings together.

I'm not sure if this package is the source of your bug.
